src/middlewared/middlewared/alert/source/deprecated_service.py

```diff
@@ -2,12 +2,20 @@
 
 from middlewared.alert.base import Alert, AlertClass, SimpleOneShotAlertClass, AlertCategory, AlertLevel
 
+URL = "https://www.truenas.com/docs/scale/scaledeprecatedfeatures/"
+
 
 class DeprecatedServiceAlertClass(AlertClass, SimpleOneShotAlertClass):
     category = AlertCategory.SHARING
     level = AlertLevel.WARNING
     title = "Deprecated Service is Running"
-    text = "The following running service is deprecated and will be removed in a future release: %(service)s"
+    text = (
+        "The following active service is deprecated in SCALE Bluefin %(service)s."
```

Comment:
By including the release names you've built-in the requirement to revisit this on the next major release.  Maybe that's by intent,   Maybe instead of 'deprecated in SCALE Bluefin' it might be 'now deprecated in SCALE'  and leave out the mention of Cobia in the next sentence.

Reply:
Valid point, I've removed mention of the specific version names.